source/is.js

```diff
@@ -3,7 +3,8 @@ import _curry2 from './internal/_curry2.js';
 
 /**
  * See if an object (`val`) is an instance of the supplied constructor. This
- * function will check up the inheritance chain, if any.
+ * function will check up the inheritance chain, if any. 
+ * If (`val`) was created using `Object.create`, `R.is(Object, val) === true`.
```

Comment:
Nit: are the parentheses necessary here?

```none
If `val`…
```

vs

```none
If (`val`)…
```

Reply:
Good catch -- probably not.

Reply:
@wiowou: If you can fix this nit, I think we're ready to merge.

Reply:
Yes, certainly.

Reply:
Probably because it was already present in `L5` https://github.com/ramda/ramda/blob/master/source/is.js#L5

Reply:
Yes, it was. However you prefer.

Reply:
I think in this case the parentheses indicated `val` as related to "object". Similar to:

```
See if an object (i.e. `val`) is an instance of the supplied constructor.
```

But here I think you probably meant just this: (wrapping `val` with parentheses didn't seem necessary to me)

```
If `val` was created using `Object.create`, `R.is(Object, val) === true`.
```

I didn't intend to block this PR. I'm happy to change this later.

Reply:
Quite right. Thank you for giving me a chance to correct this.